{
  "id": "EUVD-2019-0104",
  "enisaUuid": "1c45c0f2-08bc-34d8-9ddd-ab2eec4b51aa",
  "description": "psutil (aka python-psutil) through 5.6.5 can have a double free. This occurs because of refcount mishandling within a while or for loop that converts system data into a Python object.",
  "datePublished": "Nov 12, 2019, 2:15:00 AM",
  "dateUpdated": "Nov 18, 2019, 9:15:00 PM",
  "baseScore": 7.5,
  "baseScoreVersion": "3.1",
  "baseScoreVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
  "references": [
    "https://github.com/giampaolo/psutil/pull/1616",
    "https://lists.debian.org/debian-lts-announce/2019/11/msg00018.html",
    "https://usn.ubuntu.com/4204-1/",
    "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/2P7QI7MOTZTFXQYU23CP3RAWXCERMOAS/",
    "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/OLETTJYZL2SMBUI4Q2NGBMGPDPP54SRG/",
    "https://github.com/advisories/GHSA-qfc5-mcwq-26q8",
    "https://nvd.nist.gov/vuln/detail/CVE-2019-18874",
    "https://github.com/giampaolo/psutil/commit/7d512c8e4442a896d56505be3e78f1156f443465",
    "https://github.com/giampaolo/psutil/blob/master/HISTORY.rst#566",
    "https://github.com/pypa/advisory-database/tree/main/vulns/psutil/PYSEC-2019-41.yaml",
    "https://www.suse.com/security/cve/CVE-2019-18874.html",
    "https://access.redhat.com/errata/RHSA-2021:4324",
    "https://access.redhat.com/errata/RHSA-2021:4162",
    "https://access.redhat.com/errata/RHSA-2021:1313",
    "https://access.redhat.com/errata/RHSA-2020:5249",
    "https://access.redhat.com/errata/RHSA-2020:4299",
    "https://access.redhat.com/errata/RHSA-2020:4255",
    "https://access.redhat.com/errata/RHSA-2020:4254",
    "https://access.redhat.com/errata/RHSA-2020:2635",
    "https://access.redhat.com/errata/RHSA-2020:2593",
    "https://access.redhat.com/errata/RHSA-2020:2583",
    "https://ubuntu.com/security/CVE-2019-18874",
    "https://advisories.mageia.org/CVE-2019-18874.html",
    "https://linux.oracle.com/cve/CVE-2019-18874.html",
    "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/2P7QI7MOTZTFXQYU23CP3RAWXCERMOAS/",
    "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/OLETTJYZL2SMBUI4Q2NGBMGPDPP54SRG/"
  ],
  "aliases": [
    "CVE-2019-18874",
    "GHSA-qfc5-mcwq-26q8",
    "PYSEC-2019-41"
  ],
  "assigner": "mitre",
  "epss": 0.11,
  "enisaIdProduct": [
    {
      "id": "54c12abb-2fc2-3a5e-9c75-10e0932e76b3",
      "product": {
        "name": "n/a"
      },
      "product_version": "n/a"
    }
  ],
  "enisaIdVendor": [
    {
      "id": "cc95b251-485a-3b03-81ae-2464c976c8e2",
      "vendor": {
        "name": "n/a"
      }
    }
  ]
}